app/config.py

```diff
@@ -75,6 +75,10 @@
 # For running Open Telemetry locally, set `OTEL_DAEMON_ADDRESS`, otherwise defaults to production value.
 otel_daemon_address = os.getenv('OTEL_DAEMON_ADDRESS', 'http://127.0.0.1:4317')
 
+# Note: All the topics here use the English version of the Candidate Set Ids.
+# They then get merged with the metadata from DynamoDB, TopicMetadata table and our translation files in this repository.
+# From there it is then mapped to the real candidate set id for that language from the _TOPIC_CANDIDATE_SETS variable based on the locale model
```

Comment:
It's been a while, so I had to dive in here again. These are topic ids, not candidate set ids.

I guess at some point we started transitioning from a UUID to a more readable "corpus_topic_id". We should use the latter for readability, and refactor [topic_provider](https://github.com/Pocket/recommendation-api/blob/7e12258c8203214b68ca031fc925914ed8d75636/app/data_providers/topic_provider.py#L45-L52) to fetch a topic using for example "BUSINESS".